cinema/views.py

```diff
@@ -1 +1,68 @@
-# write views here
+from rest_framework import status, generics, mixins, viewsets
+
+from cinema.models import Actor, Genre, CinemaHall, Movie, MovieSession
+from cinema.serializers import (
+    ActorSerializer,
+    GenreSerializer,
+    CinemaHallSerializer,
+    MovieSerializer,
+    MovieListSerializer,
+    MovieSessionSerializer,
+    MovieRetrieveSerializer,
+    MovieSessionListSerializer,
+    MovieSessionRetrieveSerializer
+)
+
+
+class ActorViewSet(viewsets.ModelViewSet):
+    queryset = Actor.objects.all()
+    serializer_class = ActorSerializer
+
+
+class GenreViewsSet(viewsets.ModelViewSet):
+    queryset = Genre.objects.all()
+    serializer_class = GenreSerializer
+
+
+class CinemaHallViewSet(viewsets.ModelViewSet):
+    queryset = CinemaHall.objects.all()
+    serializer_class = CinemaHallSerializer
+
+
+class MovieViewSet(viewsets.ModelViewSet):
+    queryset = Movie.objects.all()
+    serializer_class = MovieListSerializer
+
+    def get_queryset(self):
+        queryset = self.queryset
```

Comment:
You should remove this line of code because it's redundant; the queryset is already defined at the class level, and you can directly use self.queryset in the get_queryset method.